### PR TITLE
Fixed case in classes and method names

### PR DESCRIPTION
**Pull request by demis-palma**
Fixed case in classes and method names to match the definitions.
Fixed the carriage return character on two files, so that each line there appears changed.

### CODE REVIEW

File: source/administrator/components/com_jedchecker/views/uploads/view.html.php [hunk 1]
```diff
@@ -61,19 +61,19 @@ public function setToolbar()
...
 		}
 
 		if ($this->filesExist('unzipped'))
 		{
-			JToolBarHelper::custom('police.check', 'search', 'search', 'Check', false);
+			JToolbarHelper::custom('police.check', 'search', 'search', 'Check', false);
 		}
 
-		JToolBarHelper::title('JED checker');
+		JToolbarHelper::title('JED checker');
```
Comment (JazParkyn) [inline source/administrator/components/com_jedchecker/views/uploads/view.html.php]: Jdocs state it should be JToolBarHelper

File: source/administrator/components/com_jedchecker/views/uploads/view.html.php [hunk 2]
```diff
@@ -28,7 +28,7 @@ class JedcheckerViewUploads extends JViewLegacy
 	public function display($tpl = null)
 	{
 		$this->setToolbar();
-		$this->jsOptions['url'] = JURI::base();
+		$this->jsOptions['url'] = JUri::base();
```
Comment (JazParkyn) [inline source/administrator/components/com_jedchecker/views/uploads/view.html.php]: Jdocs lists JURI capitalised: https://docs.joomla.org/JURI/root

**Comment by demis-palma** [comment]
Yes, I see.

However, I based my edit on the current class definitions, as they are in the official repository. See [JUri](https://github.com/joomla/joomla-cms/blob/staging/libraries/joomla/uri/uri.php) and [JToolbarHelper](https://github.com/joomla/joomla-cms/blob/staging/administrator/includes/toolbar.php).

JURI has been [renamed to JUri](https://docs.joomla.org/Potential_backward_compatibility_issues_in_Joomla_3_and_Joomla_Platform_12.2#JUri) in the year 2013, so that I'm pretty sure that the documentation is obsolete and it should be updated.
About JToolbarHelper I don't remember, but [the documentation itself](https://docs.joomla.org/JToolBarHelper) is contradictory, as it uses both the forms JToolBarHelper and 2 lines below JToolbarHelper. I would update this page as well.

**Comment by JazParkyn** [comment]
Agreed on both by definition in the current repo. Merging.

**Pull request merged**